hooks/ad-join-relation-joined.ps1

```diff
@@ -22,27 +22,28 @@ try {
     Import-Module JujuHooks
     Import-Module JujuUtils
 
+    $renameReboot = Rename-JujuUnit
```

Comment:
I will use OpenStack provider with the latest Juju stable version, and see if they still use long preffixes. But people Juju 1.x support is not yet dropped.